### PR TITLE
Allow reverse chronology (most recent first) order of output

### LINKED ISSUE #22: Allow history to be in reverse chronology (most recent to oldest)

### PR DESCRIPTION
**Pull request by tjchambers**
Fixes #22

### CODE REVIEW

File: spec/paper_trail_scrapbook/life_history_spec.rb [hunk 1]
```diff
@@ -120,6 +120,40 @@ def target.trailed_related_content
...
+          config.time_format = '%A, %d %b %Y at %l:%M:%S.%9N %p'
+
+          def target.trailed_related_content
+            [book, author]
+          end
+        end
+
+        after do
+          config = PaperTrailScrapbook.config
+          config.recent_first = false
```
Comment (dinjas) [inline spec/paper_trail_scrapbook/life_history_spec.rb]: little concerned that the spec still passes if this line is commented out.

Comment (dinjas) [reply spec/paper_trail_scrapbook/life_history_spec.rb]: oh, hold that thought - wrong line

File: spec/paper_trail_scrapbook/life_history_spec.rb [hunk 2]
```diff
@@ -120,6 +120,40 @@ def target.trailed_related_content
...
       end
+
+      context 'recent first' do
+        before do
+          author
+          book
+          target
+
+          config = PaperTrailScrapbook.config
+          config.recent_first = true
```
Comment (dinjas) [inline spec/paper_trail_scrapbook/life_history_spec.rb]: Ok, this is the right line - little concerned that spec passes if this is commented out.

Comment (tjchambers) [reply spec/paper_trail_scrapbook/life_history_spec.rb]: the spec is not checking the sequence - let me see if I can encourage that. the data is limited

**Comment by dinjas** [comment]
Awesome, thanks.

**Pull request merged**